platform/platform-configuration-listener/pom.xml

```diff
@@ -29,6 +29,10 @@
     </description>
     <packaging>bundle</packaging>
 
+    <properties>
```

Comment:
Was able to change the pom file and get things to work with the same powermock version.